api/src/test/resources/org/openmrs/module/mksreports/include/patientHistoryManagerTestDataset.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<dataset>
+
+	<concept concept_id="100000" retired="false" datatype_id="1"
+		class_id="5" is_set="false" creator="1"
+		date_created="2004-08-12 00:00:00.0" version="0.1" changed_by="1"
+		date_changed="2006-06-09 14:57:33.0"
+		uuid="5090AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA" />
+
+	<concept concept_id="100001" retired="false" datatype_id="1"
+		class_id="5" is_set="false" creator="1"
+		date_created="2004-08-12 00:00:00.0" version="0.1" changed_by="1"
+		date_changed="2006-06-09 14:57:33.0"
+		uuid="2190BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB" />
+
+	<concept concept_id="100002" retired="false" datatype_id="1"
+		class_id="5" is_set="false" creator="1"
+		date_created="2004-08-12 00:00:00.0" version="0.1" changed_by="1"
+		date_changed="2006-06-09 14:57:33.0"
+		uuid="2188BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB" />
+
+	<concept concept_id="100003" retired="false" datatype_id="1"
+		class_id="5" is_set="false" creator="1"
+		date_created="2004-08-12 00:00:00.0" version="0.1" changed_by="1"
+		date_changed="2006-06-09 14:57:33.0"
+		uuid="2347BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB" />
+
+	<concept_name concept_id="100000" name="Height (cm)"
+		locale="en" creator="1" date_created="2008-08-18 12:38:58.0"
+		concept_name_id="100000" voided="false"
+		uuid="5090AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA" locale_preferred="true" />
+
+	<concept_name concept_id="100001" name="Temperature (C)"
+		locale="en" creator="1" date_created="2008-08-18 12:38:58.0"
+		concept_name_id="100001" voided="false"
+		uuid="2190BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB" locale_preferred="true" />
+
+	<concept_name concept_id="100002" name="Pulse" locale="en"
+		creator="1" date_created="2008-08-18 12:38:58.0"
+		concept_name_id="100002" voided="false"
+		uuid="2188BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB" locale_preferred="true" />
+
+	<concept_name concept_id="100003" name="Pulse" locale="en"
```

Comment:
Either use another concept from Core's standard dataset, or make sure that this name is changed as "pulse" is already used.

Reply:
done